boards/xtensa/odroid_go/board_init.c

```diff
@@ -22,6 +23,10 @@ static int board_odroid_go_init(const struct device *dev)
 	gpio_pin_configure(gpio, LED_B_PIN, GPIO_OUTPUT);
 	gpio_pin_set(gpio, LED_B_PIN, 0);
 
+	/* turns LCD backlight on */
+	gpio_pin_configure(gpio, BL_PIN, GPIO_OUTPUT);
+	gpio_pin_set(gpio, BL_PIN, 1);
```

Comment:
please use the regulator infrastructure instead, see e.g. https://github.com/zephyrproject-rtos/zephyr/pull/44080

Reply:
That's a nice feature. Changes applied.